packages/api-explorer/src/form-components/MultiSchemaField.jsx

```diff
@@ -0,0 +1,118 @@
+require('../../style/main.scss');
```

Comment:
Shouldn't need to load this here fyi

Reply:
cool, just copied it from another file, will remove